src/communication/azext_communication/manual/_params.py

```diff
@@ -256,8 +256,15 @@ def _load_email_arguments(self):
         c.argument('text', options_list=['--text'], type=str, help='Plain text version of the email message. Optional.')
         c.argument('html', options_list=['--html'], type=str, help='Html version of the email message. Optional.')
         c.argument('recipients_to', options_list=['--to'], nargs='+', help='Recepients email addresses.')
+        c.argument('importance', options_list=['--importance'], arg_type=get_enum_type(['normal', 'low', 'high']),
+                   help='The importance type for the email. Known values are: high,'
+                   ' normal, and low. Default is normal. Optional')
         c.argument('recipients_cc', options_list=['--cc'], nargs='+', help='Carbon copy email addresses.')
         c.argument('recipients_bcc', options_list=['--bcc'], nargs='+', help='Blind carbon copy email addresses.')
+        c.argument('reply_to', options_list=['--reply-to'], type=str, help='Reply-to email address. Optional.')
+        c.argument('disable_tracking', options_list=['--disable-tracking'], arg_type=get_three_state_flag(),
+                   help='Indicates whether user engagement tracking should be disabled for this request if'
+                   'the resource-level user engagement tracking setting was already enabled. Optional.')
```

Comment:
Suggestion to re-write the help description as below for more clarity:
"Indicates whether user engagement tracking should be disabled for this specific request. This is only applicable if the resource-level user engagement tracking setting was already enabled in control plane."